free-compiler.cabal

```diff
@@ -170,6 +170,7 @@ library freec-internal
                       , FreeC.Pass.ExportPass
                       , FreeC.Pass.ImplicitPreludePass
                       , FreeC.Pass.ImportPass
+                      , FreeC.Pass.KindInferencePass
```

Comment:
Since the `KindInferencePass` does not actually infer any kinds, but checks whether every type actually has kind `*`, I'd call it `KindCheckPass` instead.